geppetto-showcase/README.md

```diff
@@ -8,10 +8,19 @@ Geppetto Meta showcase of components & features
 
 ## Development
 
-1. `git clone -b development https://github.com/MetaCell/geppetto-meta.git`
-2. `cd geppetto-meta/geppetto-showcase`
-4. `yarn`
-5. `yarn start`
+1. `cd geppetto.js`
+2. `yalc publish`
```

Comment:
Would add a command that installs yalc or link to: https://github.com/wclr/yalc#installation